core/templates/dev/head/pages/exploration_editor/editor_tab/state_content_editor_directive.html

```diff
@@ -1,3 +1,8 @@
+<style>
+  .oppia-state-content-display .protractor-test-state-content-display p{
```

Comment:
Note that there is already a styling defined for .oppia-state-content-display in oppia.css and it would be better to add it there itself I think. Also refrain from styling using .protractor-test-* classes as these are used by the e2e testing framework as selectors. Change this to either to style using an oppia-* tag or similar.

Reply:
I'm not defining new CSS for the classes, I'm just adding three levels of precedence for ensuring that the CSS is not applied to any other paragraph. It is applied on to the paragraph which lies under the two classes. PS: Notice that I haven't used a comma :)

Reply:
Thanks for that clarification @WickedBrat. In that case, might be better to  add a separate class and not use the protractor-test-* class that you have used. 

Now while taking a second look I realized that you have fixed this for only the state editor part of the editor window. If you try adding a multiple choice option (which is also a rich text field), the spacing is off. Maybe you have to edit in more places. Or a better solution would be to probably style all p tags under oppia-rte-editor as I guess these are the tag that you want to style?